tests/roles/mariadb_copy/templates/get_dbs_ospdo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+galera_con=$(sudo podman ps --format="{{.Names}}"|grep galera)
+mariadb_templates='/home/cloud-admin/data-plane-adoption_pkomarov/tests/roles/mariadb_copy/templates/'
+
+sudo podman cp ${mariadb_templates}/dump_dbs_ospdo.sh $galera_con:/
+sudo podman exec -it $galera_con bash -c "/dump_dbs_ospdo.sh"
+sudo podman cp  ${galera_con}:/tmp/mariadb /tmp
```

Comment:
we do not run podman on operator's machine anymore. All tasks are executed on destionation. Please see dump_dbs.bash of mariadb_copy role

Reply:
Why not to reuse  mariadb_copy role?

Reply:
+1, the shell scripts in the tests have no representation in the docs.